vmck/backends/qemu.py

```diff
@@ -73,6 +73,17 @@ def task_group(job, options):
         '-device', 'virtio-9p-pci,fsdev=vmck,mount_tag=vmck',
     ]
 
+    if options['usbstick']:
```

Comment:
Since the QEMU option name is `usb-storage`, could we use "storage" instead of "stick" for the name?

Reply:
Done.